### DISCUSSION TITLE
How to get grey colour for explanations in autocomplete menu

### DISCUSSION THREAD

**Discussion by x51mon** [marlonrichert/zsh-autocomplete#245] [Q&A] [answered]
How can I change the colour to grey for the headlines in the suggestions? Like the 'recent directory' in this screenshot from the readme

![recent dirs](https://raw.githubusercontent.com/marlonrichert/zsh-autocomplete/main/.img/recent-dirs.png)

Also how can the numbers in the history menu be changed to grey?

Sorry if these are stupid questions.

**Comment by marlonrichert** [comment, marked as answer]
No worries, it's not a stupid question. 🙂

You can get the colors from the screenshots by installing the [Z Colors plugin](https://github.com/marlonrichert/zcolors).

**Reply by x51mon** [reply]
Thank you. I will look into configuring that.
